Ghost protocol upgrade files/tool-orchestration-auth.ts

```
@@ -0,0 +1,520 @@
+          method: 'POST',
+          headers: {
+            'Authorization': clickupToken,
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            name: params.name,
+            description: params.description,
+            assignees: params.assignees || [],
+            priority: params.priority || 3,
+          }),
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)